gamePlay.js

```diff
@@ -0,0 +1,132 @@
+var fs = require("fs");
+
+var inquirer = require('inquirer');
+
+var Word = require('./Word');
+
+var Letter = require("./Letter");
+
+var prompt = require('prompt');
+ 
+var triesLeft = 10;
+
+var choosenWord;
+
+var wordArray =[];
+
+var wordString;
+
+var noCount;
+
+var guessLeft = 5;
+
+var wrongString;
+
+var wrongArray = [];
+
+var correctCount;
+
+function start(){
```

Comment:
You have too many global variables. Best practice dictates when you have functions are all interconnected it is better to just create a constructor and make all the associated functions into methods, that way your data is safer as it is contained with an instance and not just floating around the global scope.